tox.ini

```diff
@@ -11,4 +11,4 @@ commands = coverage run runtests.py
 deps =
     django18: Django>=1.8,<1.9
     django110: Django>=1.10,<1.11
-    -r{toxinidir}/requirements-test.txt
+    -rrequirements-test.txt
```

Comment:
no https://stackoverflow.com/questions/12258456/how-can-tox-install-the-modules-via-the-requirements-file#comment38660198_18964795 dunno why though